### ISSUE TITLE
Invalid PowerShell command

### ISSUE THREAD

**Issue by JFBDO2022** [MicrosoftDocs/microsoft-365-docs#12610]
The PowerShell command provided to perform the test doesn't work.  This is the same test located actually on the onboarding page in Microsoft 365 Defender, but it is an invalid test.  There's essentially no "1.exe" for it to be downloaded (or more specifically, no processes listening on port TCP 80 on the machine for it to be downloaded from).  Not sure if there used to be some capability or where it is at, but seems like others out on the Internet have similar issues/complaint with this part of the documentation.

Thanks.

PS C:\> (New-Object System.Net.WebClient).DownloadFile('http://127.0.0.1/1.exe', 'C:\\test-MDATP-test\\invoice.exe')
Exception calling "DownloadFile" with "2" argument(s): "An exception occurred during a WebClient request."
At line:1 char:1
+ (New-Object System.Net.WebClient).DownloadFile('http://127.0.0.1/1.ex ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException

PS

PS C:\> invoke-webrequest -uri 'http://127.0.0.1/1.exe' -Method get
invoke-webrequest : Unable to connect to the remote server
At line:1 char:1
+ invoke-webrequest -uri 'http://127.0.0.1/1.exe' -Method get
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

P

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1d716f82-0a0f-49b8-587e-589d166e602e
* Version Independent ID: 2c91a5fb-0567-f267-29f9-4f2660742f6a
* Content: [Run a detection test on a device to verify it has been properly onboarded to Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/run-detection-test?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/run-detection-test.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/run-detection-test.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

**Comment by msbemba** [comment]
@JFBDO2022 Thank you for your feedback. I was able to run and test this, it closes the PowerShell windows you have open. Waiting a few minutes allows that event to be registered within the portal and then it should be listed as an alert like below

![image](https://github.com/MicrosoftDocs/microsoft-365-docs/assets/45987684/b09e30ab-0884-4d59-8d65-ccde81aabd09)


If you continue to have this issue, I'd recommend contacting support via the need help section on the admin center, so that the support engineer can assist you in fixing this.

Hope this helps!

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @JFBDO2022 Hope this comment is helpful for you. Considering the nature of the issue, as advised, please open a service ticket in your tenant and follow with them for the resolution. You can contact Microsoft support via the need help section on the admin center. We proceed to close it as here we are limited to documentation fixes and improvements only.
We will be very happy if you can share the outcome or resolution with us if you see documentation update is required. We will certainly update this documentation with that valuable information. Thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by JFBDO2022** [comment]
Alright...I tested it on a few different machines, none trigger it with your PowerShell example.
EICAR works fine.  Tried on Server 2019 & Win10 with same results.  Even locally, only EICAR test triggers, not the PowerShell
Clean machines built in Azure from the Microsoft template, so not like I'm doing something wrong with my builds.
And just because the PowerShell window closes like it ought to doesn't mean that the script performed what it should have.  If you do the step manually as I showed you, there is no file to download and there are no listeners on the port you're saying to download the file from.

Local log:
***@***.***

Security Center.
***@***.***
